README.md

```diff
@@ -83,7 +83,50 @@ The CKAN core and base images can be built on Github actions by selecting the `B
 
 ## Deploying CKAN
 
-By default, merging a PR into main will trigger a build of the CKAN image and create PRs on the govuk-dgu-charts repo. Updating the tag will build and push a tagged image and create the PR for Staging and Production environments in the govuk-dgu-charts repo.
+### Environments and URLs
+
+- **Integration**: [https://ckan.integration.publishing.service.gov.uk](https://ckan.integration.publishing.service.gov.uk/)
+- **Staging**: [https://ckan.staging.publishing.service.gov.uk](https://ckan.staging.publishing.service.gov.uk//)
+- **Production**: [https://ckan.publishing.service.gov.uk/](https://ckan.publishing.service.gov.uk/)
+
+---
+
+### Deploying to Integration
+
+1. **Make your change on a branch**
+   Develop your feature or fix on a dedicated branch.
+
+2. **Have the change approved**
+   Open a Pull Request, get it reviewed and approved.
+
+3. **Merge the change into `main`**
+   Once approved, merge your branch into `main`. This triggers a GitHub Action that:
+   - Builds a new CKAN Docker image.
+   - Creates a Pull Request in the [govuk-dgu-charts](https://github.com/alphagov/govuk-dgu-charts) repo, updating the CKAN image tag.
+
+4. **Merge the newly created PR**
+   In the govuk-dgu-charts repo, merge the auto-created PR. Your changes will then be deployed to **Integration** at:
+   [https://ckan.integration.publishing.service.gov.uk/](https://ckan.integration.publishing.service.gov.uk)
+   *(Note: You’ll need to be on VPN to access Integration.)*
```

Comment:
Actually I don't think you need to be on the VPN to access any of the environments for CKAN at the moment.